docs/book/migration/to-3-0.md

```diff
@@ -0,0 +1,22 @@
+# Migration guide
+
+Migration guide for Laminas Cache version 3.0.0.
```

Comment:
In the entire documentation we use the same names like for the installation command of Composer, with one exception: "Mezzio".
```suggestion
Migration guide for laminas-cache version 3.0.0.
```